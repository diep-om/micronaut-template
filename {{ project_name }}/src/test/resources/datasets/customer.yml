customer:
  - customer_number: 1
    customer_name: Mike
    contact_last_name: Hello
    contact_first_name: There
    phone: "111111"
  - customer_number: 2
    customer_name: John
    contact_last_name: Oh
    contact_first_name: Wow
    phone: "222222"
