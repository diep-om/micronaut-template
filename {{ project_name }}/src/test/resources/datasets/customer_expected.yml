customer:
  - customer_number: 1
    customer_name: Mike
    phone: "333333"
  - customer_number: 2
    customer_name: John
    phone: "444444"
